qa/workunits/mon/crush_ops.sh

```diff
@@ -74,11 +74,16 @@ ceph osd crush rm foo
 ceph osd crush rm osd.$o2 host2
 ceph osd crush rm host2
 
+ceph osd crush add-bucket foo host
+ceph osd crush move foo root=default rack=localrack
+
+ceph osd crush create-or-move osd.$o1 1.0 root=default
+ceph osd crush move osd.$o1 host=foo
+ceph osd find osd.$o1 | grep host | grep foo
```

Comment:
Looks like this is causing rados suite tests to fail like so:

```
2017-02-13T21:59:24.604 INFO:tasks.workunit.client.0.smithi199.stderr:+ ceph osd find osd.7
2017-02-13T21:59:24.658 INFO:teuthology.orchestra.run.smithi201.stderr:2017-02-13 21:59:24.657796 7f0b5cc64700 -1 WARNING: the following dangerous and experimental features are enabled: *
2017-02-13T21:59:24.665 INFO:teuthology.orchestra.run.smithi201.stderr:2017-02-13 21:59:24.663054 7f0b5cc64700 -1 WARNING: the following dangerous and experimental features are enabled: *
2017-02-13T21:59:24.676 INFO:tasks.workunit.client.0.smithi199.stderr:2017-02-13 21:59:24.670512 7efe4a460700 -1 WARNING: the following dangerous and experimental features are enabled: *
2017-02-13T21:59:24.688 INFO:tasks.workunit.client.0.smithi199.stderr:2017-02-13 21:59:24.683239 7efe4a460700 -1 WARNING: the following dangerous and experimental features are enabled: *
2017-02-13T21:59:25.399 INFO:tasks.workunit.client.0.smithi199.stderr:Invalid command:  osd.7 doesn't represent an int
2017-02-13T21:59:25.399 INFO:tasks.workunit.client.0.smithi199.stderr:osd find <int[0-]> :  find osd <id> in the CRUSH map and show its location
2017-02-13T21:59:25.399 INFO:tasks.workunit.client.0.smithi199.stderr:Error EINVAL: invalid command
2017-02-13T21:59:25.410 INFO:tasks.workunit:Stopping ['mon/pool_ops.sh', 'mon/crush_ops.sh', 'mon/osd.sh', 'mon/caps.sh'] on client.0...
```

@shinobu-x Please try to change this line to

```
ceph osd find $o1 | grep host | grep foo
```

and add a line like this at the end of the commit message:

```
Conflicts:
    qa/workunits/mon/crush_ops.sh: adapted "ceph osd find" to jewel syntax
```